src/Components/SignIn/SignIn.tsx

```diff
@@ -1,7 +1,28 @@
 import React from 'react'
-
+import { Box, Typography, TextField, Button, Grid } from '@mui/material';
+import signin from "../../assets/bg-signin.png";
 export default function SignIn() {
   return (
-    <div>SignIn</div>
+    <>
```

Comment:
You can extract common typography styles in this file and spread them instead of writing them multiple times.
For example:
```
const commonTextStyle = {
  color: "#000000",
  fontFamily: "cairo",
  fontSize: "20px",
  fontWeight: 300,
};

<Typography variant="h4" sx={{ ...commonTextStyle, marginBottom: "20px" }}>
          Sign in
</Typography>
```